### ISSUE TITLE
keep same tab/section selecting when clicking on widgets to edit-Feature Request

### ISSUE THREAD

**Issue by rburn99** [pojome/elementor#5127]
I would like to see Elementor keep the same property tab selected when selecting elements/widgets to edit. Then when we want to for example, change a border we put around a bunch of images we can simply click on each one and then modify the appropriate parameter without having to click back to that parameter each time.
Even further, if the next selected widget has the same expandable section open, you could have the edit window remain set to showing that section expanded or property visible. For example, if you have the Advanced tab selected and the Border section expanded, you could have the Advanced tab selected and the Border section expanded when you click the next widget.

**Comment by bainternet** [comment]
Thanks for the feedback,
Please follow guidelines for opening a new feature request, so we can help you better.
This report is missing key information: 

* Prerequisites.
* Description and examples.

**Comment by rburn99** [comment]
nah... I was just trying to help you make a better product. I clicked 
create Feature Request. When I saved it I saw it did not show up like 
the others. I went back to the editor and looked everywhere but I could 
not find any check box or anything to specify it was a feature request. 
I would have assumed clicking Feature Request during the create process 
was sufficient to tell the system it was a feature request. The best I 
could do was manually add the words Feature Request to the subject line. 
If that is not sufficient then se la vie. I was only trying to help your 
product.
